view_post.php

```diff
@@ -0,0 +1,62 @@
+<html>
+
+	<head>
+		<title> View Blog Posts </title>
+		<link rel='stylesheet' href='maindesign.css' />
+	</head>
+
+	<body>
+
+<?php
+
+$sorter= array(
+	'Blog1'=>array(
+		'<h1 class="title">Halfway Done</h1>',
+		'<img src="sophsResized.jpg" alt="All the Amateur sophomores at Senior Send Off." class="photo">',
```

Comment:
The same sort of goes for this. Could you just save the src and maybe alt for each image rather than the full html, and then just plug these values in below?